scripts/aws/lambda_exec_policy.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": "arn:aws:logs:*:*:*"
```

Comment:
Consider making this policy a Jinja template as well, as the exact resource should be different for different environments. I don't think we'd ever want to allow a Lambda to put log events in any CloudWatch Logs stream it wants.